### DISCUSSION TITLE
Equivalent to [ro] mongoose in prisma

### DISCUSSION THREAD

**Discussion by l1mindr** [prisma/prisma#16999] [Q&A] [answered]
Hi
I wanted to know what is the equivalent of this query in Prisma?
```js
await User.findOne({
    $or: [{ email }, { username: email }],
})
```

**Comment by nurul3101** [comment, marked as answer]
Hello @mohamadresaaa 👋 

The equivalent in Prisma would be:
```typescript
  await prisma.user.findFirst({
    where: {
      OR: [
        {
          email: 'test@prisma.io',
        },
        {
          username: 'test@prisma.io',
        },
      ],
    },
  });
```

Here's a reference to [OR](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#or) filter.

**Reply by l1mindr** [reply]
thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
